.travis.yml

```diff
@@ -70,10 +68,11 @@ before_script:
 - export OMPI_CC
 - export OMPI_FC
 script:
+- export CLAW_SRC=$(pwd)
 - mkdir build
 - cd build
 - cmake -DCMAKE_INSTALL_PREFIX=$HOME/claw ..
-- make -j
+- make
```

Comment:
Why disabling the parallelism here?

Reply:
There seems to be some sort of race condition in the build. I haven't been able to track it down yet, because it occurs very rarely. 

It does not seem to be very useful (OMNI build does not support parallelisation and option does not affect Java build, which is not done with make),  so I disabled it for the moment. Most of the time is currently spent in preparation for tests (transformation), where parallelisation remains enabled.

Reply:
There was no race issue so far. It would be good to find what cause the problem.

Reply:
This seems to affect only the Travis build, isn't it. Is there a significant performance impact ? If not we can open an issue and continue. @clementval Are you sure this is coming from this PR and was not already present in the master ?

Reply:
No race condition as far as I know.

Reply:
seems to occurs very rarely though, and this -j here will only make a couple of second difference, I would suggest to leave it for now and open an issue

Reply:
Do you have a Travis build number where it occurs? Just curious because I haven't seen such problem.

Reply:
Outdated, "parallel" build re-enabled.